presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -171,6 +171,7 @@
     public static final String FRAGMENT_RESULT_CACHING_ENABLED = "fragment_result_caching_enabled";
     public static final String LEGACY_TYPE_COERCION_WARNING_ENABLED = "legacy_type_coercion_warning_enabled";
     public static final String INLINE_SQL_FUNCTIONS = "inline_sql_functions";
+    public static final String CHECK_ACCESS_CONTROL_AFTER_COLUMN_PRUNING = "check_access_control_after_column_pruning";
```

Comment:
The name of the session is not clear. This is describing what it does rather than what it achieves. So maybe change to "check_access_control_on_referenced_columns_only" or something along that line.

Reply:
Changed the name to "check_access_control_on_impacting_columns_only"